.github/CODEOWNERS

```diff
@@ -115,3 +115,7 @@
 /lib/internal/bootstrap/* @nodejs/startup
 /tools/code_cache/* @nodejs/startup
 /tools/snapshot/* @nodejs/startup
+
+# V8
+/deps/v8/* @nodejs/v8-update
+/tools/v8_gypfiles/* @nodejs/v8
```

Comment:
I would have thought that these should be the other way around or both should be `nodejs/v8-update`. The gyp files are our (Node.js) responsibility now -- notifying the `nodejs/v8` team would be for more generic assistance with V8.

Reply:
+1